### ISSUE TITLE
AMP error on table element missing role presentation

### ISSUE THREAD

**Issue by BenjaminEmiliani** [SAP/spartacus#15346]
**Describe the bug**
Access Assistant reports semantic error for table layouts for lacking role presentation.

**To Reproduce**
1. Create a saved cart
2. Navigate to 'Saved Carts'
3. Run Access Assistant

**Screenshots**

<img width="1268" alt="Screen Shot 2022-03-09 at 4 55 30 PM" src="https://user-images.githubusercontent.com/49131189/157542992-7e0cd75a-fe5f-47c3-991f-0c0783cdbcf1.png">

**Comment by BenjaminEmiliani** [comment]
Closing ticket, issue reported by AMP is in fact a warning that requires manual confirmation and not a strict violation
